src/plugin/browser_field.rs

```diff
@@ -18,11 +18,13 @@ impl<'a> BrowserFieldPlugin<'a> {
     }
 
     fn request_path_is_equal_alias_key_path(
-        alias_path: &PathBuf,
+        alias_path: &Path,
         info: &Info,
         extensions: &[String],
     ) -> bool {
+        let alias_path = alias_path.normalize();
         let request_path = info.to_resolved_path();
+        let request_path = request_path.normalize();
```

Comment:
it aims to make compare `Path("/a/b") == Path("/a/../a/b")` correctly.

maybe we should find a better way to compare this